frontend/src/api/contexts/config/ConfigContext.tsx

```diff
@@ -0,0 +1,26 @@
+import { createContext, useContext, useEffect, useState } from 'react'
+import { HasChildren } from '../../../util/react-types.util'
+import axios from 'axios'
+import { API_BASE_URL } from '../../../util/configs/environment.config'
+import { ConfigDto } from './types'
+
+export const ConfigContext = createContext<ConfigDto | undefined>(undefined)
+
+export const ConfigProvider = ({ children }: HasChildren) => {
+  const [config, setConfig] = useState<ConfigDto>()
+  const [loading, setLoading] = useState<boolean>(true)
+  useEffect(() => {
+    axios
+      .get<ConfigDto>(API_BASE_URL + '/api/app')
```

Comment:
Axios is preset to include the baseUrl in every query
```suggestion
      .get<ConfigDto>('/api/app')
```